view/infinite.go

```diff
@@ -0,0 +1,431 @@
+package view
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jsdelivr/globalping-cli/client"
+	"github.com/jsdelivr/globalping-cli/model"
+	"github.com/mattn/go-runewidth"
+	"github.com/pterm/pterm"
+)
+
+// Table defaults
+var (
+	colSeparator = " | "
+)
+
+func OutputInfinite(id string, ctx *model.Context) error {
+	fetcher := client.NewMeasurementsFetcher(client.ApiUrl)
+	res, err := fetcher.GetMeasurement(id)
+	if err != nil {
+		return err
+	}
+	// Probe may not have started yet
+	for len(res.Results) == 0 {
+		time.Sleep(ctx.APIMinInterval)
+		res, err = fetcher.GetMeasurement(id)
+		if err != nil {
+			return err
+		}
+	}
+
+	if ctx.Latency || ctx.JsonOutput {
+		for res.Status == model.StatusInProgress {
+			time.Sleep(ctx.APIMinInterval)
+			res, err = fetcher.GetMeasurement(res.ID)
+			if err != nil {
+				return err
+			}
+		}
+		if ctx.Latency {
+			return OutputLatency(id, res, *ctx)
+		}
+		if ctx.JsonOutput {
+			return OutputJson(id, fetcher, *ctx)
+		}
+	}
+
+	if len(res.Results) == 1 {
+		return outputSingleLocation(fetcher, res, ctx)
+	}
+	return outputMultipleLocations(fetcher, res, ctx)
+}
+
+func outputSingleLocation(
```

Comment:
Just like native ping, we should print the final stats when the user ends the ping. Basically calculate the stats the same way as in table mode and render in this form:
```
--- 1.1.1.1 ping statistics ---
6 packets transmitted, 6 received, 0% packet loss, time 5007ms
rtt min/avg/max/mdev = 3.064/3.986/6.210/1.095 ms
```